news/2025/2025-12-12-osu-world-cup-2025-finals-recap.md

```diff
@@ -0,0 +1,311 @@
+---
+layout: post
+title: "osu! World Cup 2025: Finals Recap"
+date: 2025-12-12 20:00:00 +0000
+series: world_cups
+---
+
+Excitement and anticipation were through the roof for the osu! World Cup 2025 Finals. For the first time ever, the top 6 seeds made up the last 6 of the bracket, set to face off in a match week that was unprecedented in scope. One question remained to be answered: would it live up to the hype?
+
+![](/wiki/shared/news/2025-12-12-osu-world-cup-2025-finals-recap/banner.jpg)
+
+In the winners' Finals: #1 seed Poland and #2 seed United States. The former was seeking their first Grand Final since 2017 having already achieved their first podium since that same year. The latter was looking for redemption after failing to win the tournament in 2024 for the first time in 7 years. Meanwhile, Australia, Canada, Germany, and South Korea would fight tooth and nail in the losers' bracket for the coveted final spot on the podium. All had glory in their sights, but only one team could win the two matches required to reach Grand Finals week.
```

Comment:
```suggestion
In the winners' Finals, #1 seed Poland faced off against #2 seed United States. The former was seeking their first Grand Final since 2017, having already achieved their first podium since that same year. The latter was looking for redemption after failing to win the tournament in 2024 for the first time in 7 years. Meanwhile, Australia, Canada, Germany, and South Korea would fight tooth and nail in the losers' bracket for the coveted final spot on the podium. All had glory in their sights, but only one team could win the two matches required to reach Grand Finals week.
```